src/widgets/map/ui/localMarkingListBottomSheet.tsx

```diff
@@ -109,9 +109,9 @@ const LocalMarkingList = ({
 
   if (isLoading) {
     return (
-      <MarkingList display="grid" className="gap-1">
-        {Array.from({ length: 20 }, (_, idx) => idx).map((key) => (
-          <div key={key} className="aspect-square skeleton" />
+      <MarkingList display="grid">
+        {Array.from({ length: markingList.length }).map((_, idx) => (
+          <div key={idx} className="aspect-square skeleton  mx-1 my-1" />
```

Comment:
`mx-1 my-1`를 `m-1`로 할 수 있지 않을까요?!
```suggestion
          <div key={idx} className="aspect-square skeleton  m-1" />
```